syncmaster/db/migrations/versions/2023-11-23_0000_alter_alembic_version.py

```diff
@@ -0,0 +1,37 @@
+# SPDX-FileCopyrightText: 2023-2024 MTS PJSC
```

Comment:
It would be great to split up changes to 3 PRs:
- change migrations
- change settings
- change middlewares

If it's hard to implement in this way, just resolve this comment.

Reply:
[change migrations](https://github.com/MobileTeleSystems/syncmaster/pull/115)

Reply:
[change_settings](https://github.com/MobileTeleSystems/syncmaster/pull/112)